session_1/hello_world/app/views/page/about_me.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>harish</h1>
+<p>B.tech CHEME NITK</p>
```

Comment:
@harishA3 
We need you to add an ordered list below the <p> tag. You can add your interests as an ordered list below the p tag.
You need to add any four of your interests as explained in the reference
Reference - (https://www.w3schools.com/html/html_lists_ordered.asp)

Reply:
I am so sorry. I must have edited it out while editing the html file

Reply:
Will sure do